src/metadata.ts

```diff
@@ -449,6 +449,6 @@ export const applyMetadata = async (): Promise<void> => {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     const error = e as any;
     const message = error.response?.data || error.message;
-    logger.warn('Impossible to apply metadata', message);
+    logger.warn('Impossible to apply metadata: ${message}');
```

Comment:
This only works using backticks

```suggestion
    logger.warn(`Impossible to apply metadata: ${message}`);
```

Reply:
Sorry I did not pay attention to this ^^' Thanks!